Web Editor wiki typo fix

Laarryy opened the pull request:
Added a sentence at the beginning and fixed two typos

Laarryy committed:
fix typos and clarity

Added a sentence at the beginning and fixed two typos

Andre601 commented on Web-Editor.md:
```diff
@@ -8,6 +8,8 @@ ___
 
 Editor sessions are first created by executing a command on the server.
 
+You can run different commands to open sessions with different scopes:
```
I would suggest to have it as a new line and not a new paragraph.

So
```markdown
Editor sessions are first created by executing a command on the server.  
You can run different commands to open sessions with different scopes:
```

Instead of what you have now

Tobi406 reviewed (changes requested):
Regarding the LuckPerms nodes section, it should be `priority` for prefix/suffix as per the Commands page

BrainStone commented:
![grafik](https://user-images.githubusercontent.com/4106544/78087991-1cf72f80-73c3-11ea-97ce-1d1861333f10.png)
Just gonna leave that there...

😛

Laarryy committed:
Implement suggestions

changed <weight> to <priority> for homogeneity and changed added sentence to be a new line not new paragraph

Turbotailz commented:
Poor Larry getting roasted

Turbotailz reviewed (approved):
Thank you for taking time to improve the wiki 🙌

lucko commented:
Thanks
